testnets/terra2testnet/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "terra2",
+    "assets": [
+      {
+        "description": "The native staking token of Terra.",
+        "denom_units": [
+          {
+            "denom": "uluna",
+            "exponent": 0
+          },
+          {
+            "denom": "luna",
+            "exponent": 6
+          }
+        ],
+        "base": "uluna",
+        "name": "Luna",
+        "display": "luna",
+        "symbol": "LUNA",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/terra2testnet/images/luna.svg",
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/terra2testnet/images/luna.png"
```

Comment:
These URLs don't exist. You can point them to the mainnet images?

Reply:
done!